core/jazz-ui/src/app/secondary-components/create-service/internal/create-service.component.ts

```diff
@@ -956,11 +956,11 @@ blurRegion(){
       // {
```

Comment:
remove commented code

Reply:
This is on an internal file and is currently irrelevant to oss Jazz

Reply:
same comment as before

Reply:
removing commented code